db/changelog/v-1/11-changeset-test-agent-insert-data.xml

```diff
@@ -52,6 +54,7 @@
             <column name="passed_tests" value="0" />
             <column name="failed_tests" value="0" />
             <column name="skipped_tests" value="0" />
+            <column name="sdk" value="ubuntu:latest" />
```

Comment:
Almost forgot, I've created a directory `v-2` for new changelogs. While we don't have useful data in the DB now, it will be useful to properly prepare all infrastructure, so please move your changes in a new migration in `v-2`

Reply:
Moved